src/server/routes/index.js

```diff
@@ -23,6 +23,7 @@ router.use('*', (req, res) => {
           Please enable JavaScript to view this page.</b>
         </div>
       </noscript>
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.css">
```

Comment:
(in the future, you can make a separate pull request for a change like this, because it is unrelated to the rest of the navbar work. .....unless it was required to do the navbar work.)